pkg/reconciler/v1alpha1/revision/revision.go

```diff
@@ -359,6 +359,9 @@ func (c *Reconciler) reconcile(ctx context.Context, rev *v1alpha1.Revision) erro
 		}, {
 			name: "user deployment",
 			f:    c.reconcileDeployment,
+		}, {
+			name: "image cache",
+			f:    c.reconcileImageCache,
```

Comment:
Now that you no longer need the deployment object, why place this after reconcileDeployment and not before?

Reply:
Nothing major.  Feels like getting the Deployment created slightly sooner will result in slightly faster deployments, but.... 🤷‍♂️